INSTALL.md

```diff
@@ -125,6 +125,85 @@ source.
 
 ### Windows ###
 
+Windows installation support has recently been enhanced. The new installation
+method doesn't require Cygwin or WSL and relies only a light weight git/bash
+distribution like Git-Bash, and the Intel OneAPI base distribution as well as
+the Intel OneAPI HPC Toolkit. This allows native and redistributable statically
+linked binaries to be built if you use the appropriate compiler flags. Clients
+will only require the free redistributable Intel MPI runtime.
+
+#### New and Recommended Windows Installation ####
+
+Please follow these instructions, this is the recommended way to install
+OpenCoarrays on Windows.
+
+##### Pre-requisits #####
+
+* [CMake]
+* [Git-Bash]
+* [GCC, GFortran, Unix Make]
+* [Intel OneAPI Base] and [Intel OneAPI HPC toolkit]
+
+##### Installation Procedure #####
+
+1. Ensure all the prerequisites have been installed following the directions of
+   the provider.
+2. Launch the 64-bit Intel OneAPI developer shell. This will either be in the
+   start menu, or you may need to figure out how to launch it manually. See the
+   note below for help.
+3. Ensure that cmake, git-bash, gfortran, gcc, make are available on your default
+   path. The installers should have taken care of this for you.
+5. Launch a Git-Bash bash shell from the OneAPI environment, or ensure Git-Bash's
```

Comment:
```suggestion
5. Launch a Git BASH shell from the OneAPI environment, or ensure Git-Bash's
```

Reply:
Git-Bash is referring to the software distribution, and importantly, ships both a `git-bash.exe` and a `bash.exe` (as well as a `sh.exe`). So the repetition is intentional and specific: We want `bash.exe` from within the Git-Bash (or Git for Windows) distribution, *NOT* `git-bash.exe`. In most circumstances `git-bash.exe` would be OK, but I know that if you're using ConEMU terminal emulator `git-bash.exe` causes problems whereas `bash.exe` does not. So I think this wording should stay.